packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-listener.ts

```diff
@@ -198,7 +209,8 @@ export class ApplicationListener extends BaseListener implements IApplicationLis
       stickinessCookieDuration: props.stickinessCookieDuration,
       targetGroupName: props.targetGroupName,
       targets: props.targets,
-      vpc: this.loadBalancer.vpc,
+      targetType,
```

Comment:
I believe `targetType` is autodeduced from calling `addTarget()`, no? So you shouldn't need to pass it here?

Reply:
Cool. Didn't see that. The tests are passing, so thanks :slightly_smiling_face: